src/selectors.ts

```diff
@@ -79,6 +79,11 @@ export const Components = {
         tooltip: 'group by',
         placeholder: 'Optional. Select the option to group the results',
       },
+      Interval: {
+        label: 'Interval',
+        tooltip: 'interval',
+        placeholder: 'Optional. Enter the interval to group the results',
```

Comment:
Could we expand on this by giving details on the expected string e.g.
```suggestion
        placeholder: 'Optional. Enter the interval to group the results (expected format `<number><unit>` where unit is `m` for minutes, `h` for hours, `d` for days, or `w` for weeks)',
```